js/adapt-audio.js

```diff
@@ -72,9 +72,19 @@ define([
     onPageReady: function(view) {
       if (this.audioEnabled) {
           new AudioToggleView({model:view.model});
+          this.showPrompt();
       }
     },
 
+    showPrompt: function() {
+      var pushObject = {
```

Comment:
We could do with only showing this hint once. It currently shows on revisit to every page.